app/controllers/webhooks/websubs_controller.rb

```diff
@@ -13,6 +15,19 @@ def superfeedr_ingest
 
     puts "Received #{items.count} NewsItems from SuperFeedr."
 
+    if items.count > 0
+      Rails.cache.write("news") do
+        distribute_reads(max_lag: MAX_ACCEPTABLE_REPLICATION_LAG, lag_failover: true) do
+          news_items = default_news_query
+          if default_news_query.empty?
```

Comment:
This is a nitpick but calling `default_news_query` twice would send two SQL queries, can we just do

```
news_items = default_news_query
if news_items.empty?
  news_items = backup_default_news_query
end
```

Reply:
make sense. thanks!